core/modules/menu/menu.admin.inc

```diff
@@ -291,6 +291,8 @@ function menu_overview_form_language_filter_submit($form, &$form_state) {
  * Menu callback; Build the menu link editing form.
  */
 function menu_edit_item($form, &$form_state, $type, $item, $menu) {
+  $titles = menu_get_menus();
+  backdrop_add_js(array('menu_edit_item_parents' => $titles), 'setting');
```

Comment:
Can this be added with `#attached` on the form element instead? It's minor but it helps in situations where the form is retrieved but not actually rendered.

Reply:
Fixed